spyder/widgets/waitingspinner.py

```diff
@@ -121,9 +124,10 @@ def start(self):
             self._timer.start()
             self._currentCounter = 0
 
+        self.show()
+
     def stop(self):
         self._isSpinning = False
-        self.hide()
```

Comment:
For the spinner now, instead of using `hide`, a variable to interrupt the paint of the widget is done? There was some issue that prevents to use hide maybe?

Reply:
@dalthviz the issue is that now the spinner is shown all the time either empty or spinning. The idea is to have it occupy spaced even if not spinning to avoid the toolbar widgets suddenly jumping to the left and to the right when the spinner starts or stops.

I can add this comment to explain why, to I could add a setting on the spinner to enable disable this with a setting on the spinner.

Reply:
It would be nice for the plugins to have an option to actually set the use of the spinner (where the behavior of leaving the space should be preserved) or not having it (where it should not be taking space at all). So maybe going for the spinner setting could be worthy :+1:

Reply:
Ok @dalthviz, sounds good.